.travis.yml

```diff
@@ -5,9 +5,13 @@ env:
 - TOX_ENV=py27
 - TOX_ENV=py33
 - TOX_ENV=py34
+- TOX_ENV=py27-build-theme-mkdocs
+- TOX_ENV=py34-build-theme-mkdocs
+- TOX_ENV=py27-build-theme-readthedocs
+- TOX_ENV=py34-build-theme-readthedocs
```

Comment:
So, I added these initially because the original code didn't have test coverage and it helped me figure out issues with Python 3. I also added both themes because they use Tipue slightly differently (MkDocs uses it only from a modal, ReadTheDocs has a search page).

Now test coverage is better and things seem to be working well, they could probably be removed. I'm somewhat indifferent about them being there or not. So I guess I'll delete 'em :)